docs/guide/web-animation.md

```diff
@@ -56,3 +56,23 @@ The actual refresh rate can be computed with the following formula:
 ### Remarks on the Used Technologies and Their Limitations
 
 Please refer to [this section](web-scene.md#remarks-on-the-used-technologies-and-their-limitations).
+
+### Functions
```

Comment:
I would move this new section that describes the API just after the "How to Embed a Web Animation in Your Website" instead of adding it at then end of the page.